app/sidekiq/terms_of_use/sign_up_service_updater_job.rb

```diff
@@ -33,7 +33,7 @@ def perform(user_account_uuid, version)
       @user_account_uuid = user_account_uuid
       @version = version
 
-      return unless sec_id?
+      return if !sec_id? || agreement_unchanged?
```

Comment:
Could you change the name of the `sec_id?` function to `missing_sec_id?` and flip the return values in the `sec_id?` function? (I think renaming the function and flipping the logic seems a bit easier to understand what's going on)